packages/generator/src/service-generator.spec.ts

```diff
@@ -0,0 +1,489 @@
+import { createOptions } from '../test/test-util/create-generator-options';
+import { GlobalNameFormatter } from './global-name-formatter';
+import { ServiceMapping } from './service-mapping';
+import { VdmReturnTypeCategory, VdmProperty } from './vdm-types';
+import { parseAllServices, parseService } from './service-generator';
+
+describe('service-generator', () => {
```

Comment:
As expected, we need to merge some tests (from v2/v4 folder) if the source code does not have v2/v4 structure. Having 500 lines as test is still fine, I guess we have to find a solution when this becomes huger.

Reply:
I agree. I just moved them all togehter.One thing we could do is to put the parseAll method and parseServce to a seperate file and have the tests for this file than seperate from the one of the service-generator. I think if the test files get too large this is a good indicator that the object under test does to much and should be split.